src/loaders/OpenCellLoader.ts

```diff
@@ -77,7 +77,7 @@ class OpenCellLoader extends ThreadableVolumeLoader {
 
     const w = imageInfo.atlasTileDims.x * imageInfo.volumeSize.x;
     const h = imageInfo.atlasTileDims.y * imageInfo.volumeSize.y;
-    JsonImageInfoLoader.loadVolumeAtlasData(urls, (ch, data) => onData(ch, data, [w, h]));
+    JsonImageInfoLoader.loadVolumeAtlasData(urls, (ch, data) => onData(ch, data, [[0, 255]], [w, h]));
```

Comment:
Maybe have a comment here explaining the [[0, 255]] magic numbers? Or use a constant, since it's replicated in a  few places?